.travis.yml

```diff
@@ -1,5 +1,10 @@
 language: clojure
-lein: 2.9.1
+dist: xenial
+#lein: 2.9.1
+install:
```

Comment:
Why install lein 2.9.1 manually? Does travis' `lein` setting not work correctly on ppc64le? If so it would be valuable to have a comment to that effect.

Reply:
@justinstoller ,yes lein setting not working for ppc64le,it fails to get access the path /usr/local/bin/lein,so have given manually.